appeals/api/src/server/endpoints/integrations/integrations.broadcasters/appeal.js

```diff
@@ -0,0 +1,14 @@
+import config from '#config/config.js';
+import pino from '#utils/logger.js';
+import { EventType } from '@pins/event-client';
+
+export const broadcastAppeal = async (
+	/** @type {Number} */ appealId,
+	/** @type {string} */ updateType = EventType.Update
+) => {
+	if (!config.serviceBusEnabled) {
+		return false;
+	}
+
```

Comment:
So this one doesn't currently broadcast to service bus, it just logs?

Reply:
Yes, this is a stub for the appeal data model broadcasting, as the JSON schema is WIP. I took the opportunity to remove the broadcast of the POC that was developed last December.